src/components/facet.tsx

```diff
@@ -0,0 +1,42 @@
+import {Facet as HeadlessFacet} from '@coveo/headless';
```

Comment:
That's actually true of our live example as well, and the ui-kit sample that I took this from... since we get the values from the state, I would think that would be something handled by Headless itself. Do you see an easy way to do this? Or is this a functionality lacking in Headless at the moment?